modules/codemod/lib/v5/index.ts

```diff
@@ -1,192 +1,22 @@
-    };
-  }
-  newImports[newSource].namedImports?.push(specifier);
-};
-
-export default function transformer(file: FileInfo, api: API) {
-  const j = api.jscodeshift;
-
-  return j(file.source)
-    .find(j.ImportDeclaration)
-    .forEach(_import => {
-      // Move on if source isn't a canvas kit package
-      if (
-        typeof _import.value.source.value !== 'string' ||
-        (_import.value.source.value !== '@workday/canvas-kit-react' &&
-          !Object.keys(sourceMap).includes(_import.value.source.value))
-      ) {
-        return;
-      }
-
-      const source = _import.value.source.value;
-      const newImports: ImportReplacements = {};
-
-      _import.value.specifiers?.forEach(specifier => {
-        switch (specifier.type) {
-          case 'ImportDefaultSpecifier':
-            addMappedDefaultSpecifier(specifier, source, newImports, j);
-            break;
-          case 'ImportNamespaceSpecifier':
-            addMappedNamespaceSpecifier(specifier, source, newImports);
-            break;
-          case 'ImportSpecifier':
-          default:
-            addMappedSpecifier(specifier, source, newImports);
-            break;
-        }
-      });
-
-      if (!Object.keys(newImports).length) {
-        return;
-      }
-
-      j(_import).replaceWith(
-        Object.keys(newImports).map(source => {
-          const newImportSet: (
-            | ImportDefaultSpecifier
-            | ImportNamespaceSpecifier
-            | ImportSpecifier
-          )[] = [];
-
-          const {default: defaultImport, namespace, namedImports} = newImports[source];
-
-          // add `import X` default specifier
-          if (defaultImport) {
-            newImportSet.push(defaultImport);
-          }
-          // add `import * as X` namespace specifier
-          if (namespace) {
-            newImportSet.push(namespace);
-          }
-          // Add `import { X }` named specifier
-          if (namedImports?.length) {
-            namedImports?.forEach(specifier => {
-              newImportSet.push(specifier);
-            });
-          }
-
-          // Create new declaration based on set of imports
-          // (eg. `import X, { Y } from 'Z'` or `import * as X, { Y } from 'Z'`)
-          return j.importDeclaration(newImportSet, j.stringLiteral(source), 'value');
-        })
-      );
-    })
-    .toSource();
+import slashImports from './slashImports';
+import removeDefaultImports from './removeDefaultImports';
+import removeButtonEnums from './removeButtonEnums';
+import renameButtonRefs from './renameButtonRefs';
+
+import {API, FileInfo, Options} from 'jscodeshift';
+
+export default function transformer(file: FileInfo, api: API, options: Options) {
+  // These will run in order. If your transform depends on others, place yours after dependent
+  // transforms
+  const fixes = [slashImports, removeDefaultImports, removeButtonEnums, renameButtonRefs];
+
+  let src = file.source;
+  fixes.forEach(fix => {
```

Comment:
Consider `reduce` for this but that's a completely optional suggestion.

Reply:
That's fair. In general, I'm not happy with iteration support in Typescript. `Object.keys` give `string` as an item type and building objects in a `reduce` creates a lot of type overriding. This case will work well for a `reduce` though.